Marble Blast Platinum/platinum/client/scripts/loadinggui.cs

```diff
@@ -288,17 +289,19 @@
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Nest Eggs in PlatinumQuest are generally much more difficult to find and collect over their Easter Egg counterparts in Platinum and Ultra.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Nest Eggs in PlatinumQuest are generally much more difficult to find and collect over their Easter Egg counterparts in the other games.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: You can use a Super Speed to slow down, too. Just activate it while facing away from where you're going!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: In PlatinumQuest, respawning on a Checkpoint will cancel a currently-active Time Travel's effects.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you're new, the recommended order to play the games is: Gold, Ultra, PlatinumQuest and Platinum.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: In PlatinumQuest, unlike other games, respawning on a Checkpoint will cancel a currently-active Time Travel's effects.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you're new, the recommended order to play the games is: Gold, Ultra, PlatinumQuest, then Platinum.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Watching others play may give you tips on how to solve certain challenges.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: All but Orange-banded cannons allow you to aim them and fire at will, at varying power levels. Orange-banded cannons fire the marble instantly in the direction it's facing.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Fireball Powerup fizzles out when you touch water or collect a Bubble Powerup. Likewise, the Bubble Powerup pops away when you pick up a Fireball Powerup.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Bubble Powerup stays with you at all times until it is used up or you pick up a Fireball Powerup.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Sundial Item acts the same way as the Time Travel, it's just a bit fancier.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: PlatinumQuest Time Travel Items have identifiable bands; gold bands freeze the clock and red bands add time.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: PlatinumQuest's Time Travel Items have identifiable bands; gold bands freeze the clock and red bands add time.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Marble Blast Gold and Platinum's Time Travel Items can be identified by their sand color; yellow sands freeze the clock and red sands add time.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: In Multiplayer, if you activate a Mega Marble and hit an Ice Shard, your marble will shrink but remain unfrozen.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Colliding with an Ice Shard as a Mega Marble will shrink your marble to its regular size instead of freezing you.";
```

Comment:
This seems very similar to the tip above, just not multiplayer-specific. Is it meant to replace it?